src/github_runner_image_builder/builder.py

```diff
@@ -564,6 +570,13 @@ def _disable_unattended_upgrades() -> None:
         raise UnattendedUpgradeDisableError from exc
 
 
+def _enable_network_fair_queuing_congestion() -> None:
+    """Enable bbr traffic congestion algorithm."""
+    with open(SYSCTL_CONF_PATH, mode="a", encoding="utf-8") as sysctl_file:
+        sysctl_file.write("net.core.default_qdisc=fq\n")
```

Comment:
Do you need a leading newline here, or is it safe to assume that the file already has a trailing newline?

Reply:
I've tested without the newline, it will not append the newline and will corrupt the file :) so it is necessary here.

Reply:
@yanksyoon I was more referring to the case where the file does not currently have a new line at the end, so my question is whether you need something like `sysctl_file.write("\nnet.core.default_qdisc=fq\n")`.